assets/javascripts/render.js

```diff
@@ -168,7 +168,7 @@ function renderModuleRow(module, snippets) {
       if (title === 'wait_serial') {
         const previewLimit = 120;
         // jshint ignore:start
-        let shortText = textData.replace(/.*# Result:\n*/s, '');
+        let shortText = textData ? textData.replace(/.*# Result:\n*/s, '') : '';
```

Comment:
```suggestion
        let shortText = typeof textData === 'string' ? textData.replace(/.*# Result:\n*/s, '') : '';
```

Reply:
More explicit and safe, thanks for the suggestion!